Sources/DateRangePicker/Resources/Base.lproj/ExpandedDateRangePickerController.xib

```diff
@@ -50,7 +50,7 @@
                                         <binding destination="-2" name="maxValue" keyPath="maxDate" id="z8H-Ip-gnA"/>
                                     </connections>
                                 </datePicker>
-                                <datePicker verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="HSY-I2-CQn" customClass="DoubleClickDateRangePicker" customModule="DateRangePicker" customModuleProvider="target">
```

Comment:
Curiously, module inference needs to be disabled, it seems, when Nibs are loaded from Packages:
<img width="253" alt="2023-06-08 10-22-32 Xcode - Analytics — ExpandedDateRangePickerController xib@2x" src="https://github.com/Timing-GmbH/DateRangePicker/assets/59080/d452de04-6667-43da-af29-7f28437f867e">